src/utils.ts

```diff
@@ -95,3 +95,42 @@ export function createJavaScriptFromObject(obj_: any, intend = false) {
     }
     return final.join("\n");
 }
+
+export type Tree = Record<string, Function | any> | undefined | null;
```

Comment:
`Function` should never be used for type function since it allows function like objects.
use `()=>any` or something like it instead